tests/controllers/rootController/rootController.test.ts

```diff
@@ -7,13 +7,42 @@ describe('Root controller', () => {
 
   beforeEach(() => {
     app = express();
+    app.use(express.json());
   });
 
   it('returns useragent when /api queried', async () => {
     init(app);
     const response = await request(app).get('/api').set('User-Agent', 'tester');
     expect(response.statusCode).toEqual(200);
-    expect(response.body).toEqual({message: 'Hello: tester'});
+    expect(response.body).toEqual({ message: 'Hello: tester' });
+  });
+
+  it('/api/submit returns 200 when a valid youtube url is provided', async () => {
+    init(app);
+    await request(app).post('/api/submit')
+      .send({ url: 'https://www.youtube.com/watch?v=7rqEB4xNZU4' })
+      .set('Content-Type', 'application/json')
+      .set('Accept', 'application/json')
+      .expect(200)
+      .then((response) => {
+        expect(response).toBeDefined();
+      });
```

Comment:
Please will you match the above pattern where you first do `const reponse = await ...` and then do `expect(response.statusCode).toEqual(200)` in this test and the below tests. It just makes the tests easier to read because it's then clearer what we're actually asserting. Like with `.expect(200)` it's not really clear that the response status code is what's being tested (or even if that's what expect is actually doing here).